noir-projects/aztec-nr/aztec/src/oracle/pxe_store.nr

```diff
@@ -0,0 +1,16 @@
+#[oracle(store)]
+unconstrained fn store_oracle<let N: u32>(key: Field, values: [Field; N]) {}
+
+/// Store an array of values in local PXE database. The data is scoped to the current contract.
+pub unconstrained fn store<let N: u32>(key: Field, values: [Field; N]) {
+    store_oracle(key, values);
+}
```

Comment:
We should mention what this does if there is a value already stored (and have a test for it).

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/10867/commits/127f26d64625f62b0ec07cac2b0821a12e159f85